server/lib/src/entry.rs

```diff
@@ -29,12 +29,25 @@ pub use std::collections::BTreeSet as Set;
 use std::collections::{BTreeMap as Map, BTreeMap, BTreeSet};
 use std::sync::Arc;
 
+use crate::be::dbentry::{DbEntry, DbEntryVers};
+use crate::be::dbvalue::DbValueSetV2;
+use crate::be::{IdxKey, IdxSlope};
+use crate::credential::apppwd::ApplicationPassword;
+use crate::credential::Credential;
```

Comment:
Was this cargo fmt re-arranging things? Or your editor?

Reply:
Should be cargo fmt's doings.
with toolchain: stable-x86_64-unknown-linux-gnu (default)
rustc 1.81.0